samplesheets/tests/test_views_taskflow.py

```diff
@@ -1399,45 +1399,6 @@ def test_accept_no_request(self):
             )
         self.assertEqual(response.status_code, 404)
 
-    def test_accept_invalid_form_data(self):
```

Comment:
You probably don't want to remove this test, maybe update it to match the current implementation.